File: Source/CircleProgressBarCs/MainForm.cs
using System;
using System.Drawing;
using System.Windows.Forms;

namespace CircleProgressBarCs
{
	public partial class MainForm : Form
	{
		public MainForm()
		{
			InitializeComponent();
		}

		private void MainForm_Load(object sender, EventArgs e)
		{
#if (!DEBUG)
			this.Text = Application.ProductName;
#endif

			// Set font of CircleProgressBar.
			circleProgressBar_Target.Num.Font = new Font("Arial", 80, FontStyle.Bold, GraphicsUnit.Pixel);
			circleProgressBar_Target.Unit.Font = new Font("Arial", 40, FontStyle.Bold, GraphicsUnit.Pixel);
		}

		private void trackBar_Perc_Scroll(object sender, EventArgs e)
		{
			circleProgressBar_Target.Value = Convert.ToInt32((double)trackBar_Perc.Value / trackBar_Perc.Maximum * 100);
		}
	}
}

File: Source/CircleProgressBarVb/MainForm.vb
Public Class MainForm

	Private Sub MainForm_Load(sender As Object, e As EventArgs) Handles MyBase.Load
#If (Not Debug) Then
		Me.Text = Application.ProductName
#End If

		'Set font of CircleProgressBar.
		CircleProgressBar_Target.Num.Font = New Font("Arial", 80, FontStyle.Bold, GraphicsUnit.Pixel)
		CircleProgressBar_Target.Unit.Font = New Font("Arial", 40, FontStyle.Bold, GraphicsUnit.Pixel)
	End Sub

	Private Sub TrackBar_Perc_Scroll(sender As Object, e As EventArgs) Handles TrackBar_Perc.Scroll
		CircleProgressBar_Target.Value = Convert.ToInt32(TrackBar_Perc.Value / TrackBar_Perc.Maximum * 100)
	End Sub

End Class